### DISCUSSION TITLE
Design do produto

### DISCUSSION THREAD

**Discussion by kevinDsousa** [EasyRoomCompany/Guardian#132] [Ideas]
Boa tarde a todos, como havia comentado informalmente, mas creio que seja um assunto relevante neste momento, precisaremos de um design do produto para mantermos o padão em toda aplicação.

Se alguem sober mexer no figma ou conheça alguem que saíba, vamos precisar dessa ajuda para servir de guia para o desenvolvimento.

**Alguem tem alguma ideia que possa acrescentar nesta discussion??**

**Comment by mikaelmonteirodev** [comment]
Definido na reunião do dia 10/06 que o @AntonioCaiodev apresentará o Figma da aplicação no dia 08/07.
